catroid/src/main/java/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -1083,6 +1083,14 @@ public boolean isSingleNumberFormula() {
 		return false;
 	}
 
+	public boolean isElementANumber() {
```

Comment:
please just call it `isNumber()` or something like this. you don't need to remind the element that it is an element.

think of it this way: the call is most likely going to be something like `someElement.isElementANumber()` so IMO `someElement.isNumber()` is cleaner.

Reply:
btw: why do you need the function? there is literally the same function right above it? couln't you have reused (and renamed) that one?

Reply:
one more thing: there should be a test for this. there might already be one, since the method already exists. maybe have a look at it and see if it checks all cases that should be covered.